### ISSUE TITLE
torch.mean(input, dim=[2, 3], keepdim=True)  dim (int) – the dimension to reduce?

### ISSUE THREAD

**Issue by lmobot** [znxlwm/UGATIT-pytorch#37]
in networks.py 165 torch.mean(input, dim=[2, 3], keepdim=True) , why use list to dim

**Comment by kouhinn** [comment]
Same question for me.

**Comment by dxm343488059** [comment]
Same question for me.

**Comment by xinzi2018** [comment]
Same question for me.

**Comment by BraveAsta** [comment]
I think the function if it may be GAP(Group Average Pooling),which squeeze a feature map into an average value of it.
